.babelrc.js

```diff
@@ -0,0 +1,13 @@
+const { NODE_ENV, BABEL_ENV } = process.env;
+
+const cjs = BABEL_ENV === 'cjs' || NODE_ENV === 'test';
```

Comment:
this makes cjs transform an opt-in behaviour, rather an opt out - which is believe is safer choice, because if you dont opt-in into it your tools etc will fail fast and that will have you to explicitly opt-in into that transform

when you have to opt out of it it makes it easy to end up with deoptimized builds etc if a tool (i.e. webpack) knows how to handle both esm and cjs (in this situation it would gladly accept cjs input without any warnings that ESM format can be used)